applications/smart-distancing/eval/README.md

```diff
@@ -0,0 +1,83 @@
+
```

Comment:
really a long path for evaluation of a model :)
maybe we should shorten this path later.

Reply:
It is just work for the Town Center dataset, what should user do if he wants to evaluate a model with different dataset?

Reply:
Yeah. I've figured out some solutions which we can apply them for another time.